Automattic/wp-calypso
# Free to paid upsell nudges

This is an a/b test to determine the impact of upgrade nudges for free to paid plans.

One of two variants will appear for qualifying users:

A notice appearing in the sidebar for a site (50% of users).

<img alt="sidebar variant" src="https://cldup.com/DuLzQLaPsV.png">

To qualify for the a/b test, users must:

* have registered within with last 2 to 30 days
* be able to manage options for the current site
* have a free plan for the current site
* have no domain mapping for the current site

The a/b test name is `freeToPaidUpsell`.

# Steps for testing

You will either need a user that meets the criteria (user registered between 2 and 30 days ago) or can temporarily alter the selector function in `client/state/selectors/eligible-for-free-to-paid-upsell.js` to always return true.

With `freeToPaidUpsell` set to `sidebar`, you should see the upsell in the sidebar when managing a site.

With `freeToPaidUpsell` set to `disabled`, you should not see any the upsell messages.
